src/mtcp/mtcp_restart.c

```diff
@@ -161,6 +157,9 @@ main(int argc, char *argv[], char **environ)
     if (mtcp_strcmp(argv[0], "--use-gdb") == 0) {
       rinfo.use_gdb = 1;
       shift;
+    } else if (mtcp_strcmp(argv[0], "--mpi") == 0) {
+      runMpiProxy = 1;
```

Comment:
@rohgarg had chose `runMpiProxy` when we were still thinking about using a proxy approach for MANA.  In the end, MANA switched to using a split process approach.

MANA still creates a proxy child process on launch, but it then "copies the bits" from the child to the lower half of the parent process.

Can we change `runMpiProxy` perhaps to `launchMPI` ?  The name is motivated by the fact that DMTCP will then launch a trivial MPI application in a child process, which is destined to eventually become the lower half.

@rohgarg, what's your opinion?

Reply:
Changed to mpiMode.